.github/workflows/wasm-automatic-release-and-publish.yml

```diff
@@ -21,15 +21,48 @@ jobs:
     secrets:
       GPG_PRIVATE_KEY: ${{ secrets.GPG_PRIVATE_KEY }}
       GPG_PASSPHRASE: ${{ secrets.GPG_PASSPHRASE }}
-  release:
-    runs-on: ubuntu-latest
+
+  build-wasm: 
     needs: call-create-release-workflow
     if: ${{ needs.call-create-release-workflow.outputs.is-release }}
+    # owner/repository of workflow has to be static, see https://github.community/t/env-variables-in-uses/17466
+    uses: iotaledger/identity.rs/.github/workflows/shared-build-wasm.yml@chore/unifiy-ci #TODO change to dev
+    with:
+      output-artifact-name: identity-wasm-bindings-build
+
+  release-wasm:
+    if: ${{ needs.call-create-release-workflow.outputs.is-release }}
+    runs-on: ubuntu-latest
+    needs: build-wasm
     steps:
       - name: Checkout
         uses: actions/checkout@v2
       - name: Release to npm
         uses: './.github/actions/publish/publish-wasm'
+        with:
+          input-artifact-name: identity-wasm-bindings-build
+          npm-token: ${{ secrets.NPM_TOKEN }}
+          tag: ${{ needs.call-create-release-workflow.outputs.is-dev-release && 'dev' }}
+
+  build-stronghold-nodejs:
+    # we need to wait for the wasm publish so the peer dependency version becomes available
+    needs: release-wasm
+    if: ${{ needs.call-create-release-workflow.outputs.is-release }}
+    # owner/repository of workflow has to be static, see https://github.community/t/env-variables-in-uses/17466
+    uses: iotaledger/identity.rs/.github/workflows/shared-build-and-test-stronghold-nodejs.yml@chore/unifiy-ci #TODO change to dev
+    with:
+      input-artifact-name: identity-wasm-bindings-build
+      upload-binaries-as-artifacts: true
```

Comment:
We should add a condition to only build and not run tests, since examples can still be brittle due to network/node outages.